test/testscript.go

```diff
@@ -0,0 +1,155 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"text/template"
+
+	"github.com/rogpeppe/go-internal/testscript"
+)
+
+type ProjectTest struct {
+	t          *testing.T
+	params     testscript.Params
+	setupFuncs []func(*testscript.Env) error
+	projectDir string
+}
+
+func NewProjectTest(t *testing.T, projectDir string) *ProjectTest {
+	params := testscript.Params{
+		Files:               []string{},
+		Setup:               nil,
+		RequireExplicitExec: true,
+		RequireUniqueNames:  true,
+	}
+	return &ProjectTest{
+		t:          t,
+		params:     params,
+		projectDir: projectDir,
+	}
+}
+
+func (e *ProjectTest) CopyFile(srcRelPath string) *ProjectTest {
+	setupFunc := func(env *testscript.Env) error {
+		src := filepath.Join(e.projectDir, srcRelPath)
+		dst := filepath.Join(env.WorkDir, srcRelPath)
```

Comment:
for now, we recreate the given path from the project dir to the testscript workdir